[2024-09-25T08:51:16.256+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T08:51:16.281+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.dellstore_db.load.customers scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T08:51:16.292+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.dellstore_db.load.customers scheduled__2024-09-24T00:00:00+00:00 [queued]>
[2024-09-25T08:51:16.293+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T08:51:16.310+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): dellstore_db.load.customers> on 2024-09-24 00:00:00+00:00
[2024-09-25T08:51:16.318+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=371) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T08:51:16.320+0000] {standard_task_runner.py:72} INFO - Started process 373 to run task
[2024-09-25T08:51:16.321+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'dellstore_db.load.customers', 'scheduled__2024-09-24T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpwbopj302']
[2024-09-25T08:51:16.324+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask dellstore_db.load.customers
[2024-09-25T08:51:16.384+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.dellstore_db.load.customers scheduled__2024-09-24T00:00:00+00:00 [running]> on host 857abb7f2bfc
[2024-09-25T08:51:16.494+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='dellstore_db.load.customers' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:00:00+00:00'
[2024-09-25T08:51:16.496+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T08:51:16.512+0000] {base.py:84} INFO - Retrieving connection 'staging_db'
[2024-09-25T08:51:16.523+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T08:51:32.938+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T08:51:32.957+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T08:51:32.958+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=dellstore_db.load.customers, run_id=scheduled__2024-09-24T00:00:00+00:00, execution_date=20240924T000000, start_date=20240925T085116, end_date=20240925T085132
[2024-09-25T08:51:33.028+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T08:51:33.086+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-25T08:51:33.090+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
